department-of-veterans-affairs/caseflow
# Docket Switch Send to Judge - Make summary field required

Resolves [CASEFLOW-1056](https://vajira.max.gov/browse/CASEFLOW-1056)

### Description
Make summary field required 

### Acceptance Criteria

### Testing Plan
1. Switch to COB user
2. Add mail test 
3. Fill in all fields except summary
3. Verify submit button is not enabled 

### User Facing Changes

 BEFORE|AFTER
 ---|---

<img width="1168" alt="Screen Shot 2021-02-26 at 12 22 54 PM" src="https://user-images.githubusercontent.com/68879427/109353010-a19b3980-7830-11eb-87c7-76cf581a4222.png">
